prism/src/main/java/org/apache/falcon/resource/extensions/ExtensionManager.java

```diff
@@ -74,6 +79,33 @@
     public static final String DESCENDING_SORT_ORDER = "desc";
 
     private Extension extension = new Extension();
+    private static final ExtensionStore STORE = ExtensionStore.get();
+    private static final String RESULTS = "extensions";
+    private static final String TOTAL_RESULTS = "totalResults";
+    private static final String README = "README";
+    private static final String EXTENSION_PROPERTY_JSON_SUFFIX = "-properties.json";
+    private static final String SHORT_DESCRIPTION = "shortDescription";
+
+    private static final String TRUSTED_EXTENSION = "Trusted extension";
+    private static final String CUSTOM_EXTENSION = "Custom extension";
+
+    /**
+     * Enum for extension element.
+     */
+    private static enum ExtensionElement {
```

Comment:
The elements in this enum class are used as string constant. Looks better to just define them as constant strings.